samples/csharp_dotnetcore/48.customQABot-all-features/README.md

```diff
@@ -0,0 +1,175 @@
+# Custom Question Answering
+
+Bot Framework v4 Custom question answering bot sample. This sample demonstrates usage of advanced features of Custom question answering like [Precise answering][PA], support for unstructured sources along with [Multi-turn conversations][MT] and [Active Learning][AL] in a bot.
+
+This bot has been created using [Bot Framework][BF], it shows how to create a bot that uses the [Custom question answering feature in Language Service][LS].
+
+## Concepts introduced in this sample
+The [Custom question answering feature in Language Service][LS] enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured and unstructured documents or editorial content in minutes. In this sample, we demonstrate:
+- How to use the Active Learning to generate suggestions for knowledge base.
+- How to use follow up prompts to create multiple turns of a conversation.
+- How to configure display of precise answers.
+- How to enable/disable querying unstructured sources with the bot.
+
+## Prerequisites
+- This project requires a [Language resource](https://aka.ms/create-language-resource) with Custom question answering enabled.
+
```

Comment:
See the feedback in samples/csharp_dotnetcore/12.customQABot/README.md for lines 17-61

Reply:
Content looks OK when rendered in github, am I missing something?

https://github.com/microsoft/BotBuilder-Samples/tree/4.16-Samples/samples/csharp_dotnetcore/48.customQABot-all-features